package com.example.cqrs_challenge;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class CqrsChallengeApplication {

	public static void main(String[] args) {
		SpringApplication.run(CqrsChallengeApplication.class, args);
	}

}